lib/std/child_process.zig

```diff
@@ -174,7 +177,10 @@ pub const ChildProcess = struct {
             self.cleanupStreams();
             return term;
         }
-        try os.kill(self.pid, os.SIG.TERM);
+        os.kill(self.pid, os.SIG.TERM) catch |err| switch (err) {
+            error.ProcessNotFound => return error.AlreadyTerminated,
```

Comment:
same reason here.

Reply:
I'm less familiar with the details here, but my understanding is that until a parent process call's posix `wait`, the process (and PID) must stay around (in the zombie state if it exits normally) so the only way this would not indicate that the process has terminated normally, would be if the parent process called `wait` manually outside of `ChildProcess.wait()` (since in that case the termination status would be cached and we wouldn't try to kill the process at all)